utils/classify.py

```diff
@@ -42,12 +56,19 @@ def matcher(df, substring, column, category):
     return df
 
 
-def classify_individuals(individuals_df):
+def classify_individuals(individuals_df: pd.DataFrame):
     """Part of the classification pipeline
 
     We apply the matcher function to the individuals dataframe
```

Comment:
what I think you are doing is checking if an individual works at a fossil fuel company -- not entirely clear from the docstring...